src/funfuzz/js/shell_flags.py

```diff
@@ -333,21 +333,19 @@ def basic_flag_sets(shell_path):
     """
     basic_flags = [
         # Parts of this flag permutation come from:
-        # https://hg.mozilla.org/mozilla-central/file/c91249f41e37/js/src/tests/lib/tests.py#l13
+        # https://hg.mozilla.org/mozilla-central/file/afdeb0288690/js/src/tests/lib/tests.py#l10
         # compare_jit uses the following first flag set as the sole baseline when fuzzing
         ["--fuzzing-safe", "--no-threads", "--ion-eager"],
         ["--fuzzing-safe"],
         ["--fuzzing-safe", "--ion-offthread-compile=off", "--ion-eager"],
         ["--fuzzing-safe", "--ion-offthread-compile=off"],
         ["--fuzzing-safe", "--baseline-eager", "--no-ion"],  # This combo seems to find more issues than w/o --no-ion
         ["--fuzzing-safe", "--no-baseline", "--no-ion"],
+        ["--fuzzing-safe", "--no-baseline", "--no-asmjs", "--no-native-regexp"],  # --no-wasm cannot be in compare_jit
```

Comment:
Your comment is referencing removed code. It probably won't be helpful in the future.